### PR TITLE
Warning: get_parameter_value() is deprecated.

### PR DESCRIPTION
**Pull request by fujitatomoya**
address https://github.com/ros2/ros2cli/issues/865

### CODE REVIEW

**Comment by fujitatomoya** [comment]
CC: @iuhilnehc-ynos @Barry-Xu-2018

File: ros2param/ros2param/api/__init__.py [hunk 1]
```diff
@@ -13,23 +13,15 @@
...
 from rcl_interfaces.msg import ParameterType
 import rclpy
-from rclpy.parameter import get_parameter_value as rclpy_get_parameter_value
 from rclpy.parameter import parameter_dict_from_yaml_file
 from rclpy.parameter import parameter_value_to_python
 from rclpy.parameter_client import AsyncParameterClient
 from ros2cli.node.direct import DirectNode
 
 
-def get_parameter_value(*, string_value):
```
Comment (fujitatomoya) [inline ros2param/ros2param/api/__init__.py:27]: this is imported by `ros2component`, addressed with https://github.com/ros2/ros2cli/pull/866/commits/bc80ce8de31c4323650d5b17beba117977a16b98

**Review by clalancette — APPROVED** [review]
Looks reasonable to me.

I'll suggest running full CI with this one, just to make sure there aren't any other uses of `get_parameter_value` throughout the codebase.

**Comment by fujitatomoya** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/19712/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/14222/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/20441/)

**Pull request merged**